walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,87 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.PriorityQueue;
+import java.util.Set;
 
 public final class FindMeetingQuery {
```

Comment:
lets try to reorder the functions in this class according to chapter 5 of clean code: which is to minimize the "vertical distance" between caller and callees

"If one function calls another, they should be vertically close, and the caller should be above the callee, if at all possible"

Reply:
https://softwareengineering.stackexchange.com/questions/199311/most-human-friendly-way-to-order-class-method-definitions

Reply:
Thanks for the suggestion!!